modules/logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,197 @@
+  namespace: openshift-operators-redhat <1>
+spec:
+  channel: stable <2>
+  name: loki-operator
+  source: redhat-operators <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Create a LokiStack instance:
+
+.. Create an instance object YAML file (for example, `logging-loki.yaml`) for the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+include::snippets/logging-lokistack-cr-snip.adoc[]
+
+
+. Install the `Red Hat OpenShift Logging` Operator by creating the following objects:
+
+.. Create an Operator Group object YAML file (for example, `olo-og.yaml`) for the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+<1> You must specify the `openshift-logging` namespace.
+
+
+.. Create a Subscription object YAML file (for example, `olo-sub.yaml`) to
+subscribe a namespace to the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable # <2>
+  name: cluster-logging
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
+
+.. Verify the Operator installation.
++
+There should be a Red Hat OpenShift Logging Operator in the `openshift-logging` namespace. The Version number might be different than what is shown.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            DISPLAY                            VERSION   REPLACES                        PHASE
+cluster-logging.v5.7.4          Red Hat OpenShift Logging          5.7.4     cluster-logging.v5.7.3          Succeeded
+----
+
+. Create an OpenShift Logging instance:
```

Comment:
We should create lokistack instance before clusterlogging.  The sequence:
1. Deploy loki-operator
2. Deploy cluster-logging operator
3. Deploy lokistack under openshift-logging namespace
4. Deploy clusterlogging under openshift-logging namespace to send logs to lokistack.